test_unstructured/partition/test_email.py

```diff
@@ -110,24 +119,17 @@
 
 
 def test_partition_email_from_filename():
-    filename = os.path.join(EXAMPLE_DOCS_DIRECTORY, "fake-email.eml")
-    elements = partition_email(filename=filename)
+    elements = partition_email(filename=example_doc_path("eml/fake-email.eml"))
```

Comment:
Just for future reference, these are more compact and easier to scan if you remove the `filename=` bit. `filename` is always the first parameter and can be provided positionally. Since this form appears so often in tests I think it improves readability if it's reduced to "partition(this)" :)

Reply:
Thanks. Didn't think about that, but will try to remember to do so :)